changelog/next/breaking-changes/4336--python-uv.md

```diff
@@ -0,0 +1,4 @@
+The python operator no longer uses `pip` but rather
+[`uv`](https://github.com/astral-sh/uv). In case you set custom environment
+variables for `pip` you need to exchange those with alternative settings that
+work with `uv`.
```

Comment:
```suggestion
The `python` operator no longer uses `pip` but rather
[`uv`](https://github.com/astral-sh/uv). In case you set custom environment
variables for `pip` you need to exchange those with alternative settings that
work with `uv`.
```